src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        String color = colorSupplier.getRandomColor();
+        Random randomNumberSupplier = new Random();

```

Comment:
Don't create several instances of the same class if you can use only one instance for your purposes. The `Random` instance should be a class-level field instead of being created every time the `getRandomBall` method is called.